src/Umbraco.Tests.Integration/Services/ContentTypeServiceTests.cs

```diff
@@ -1200,7 +1200,7 @@ public void Can_Rename_PropertyGroup_With_Inherited_PropertyGroups()
             // create 'page' content type with a 'Content_' group
             var template = TemplateBuilder.CreateTextPageTemplate();
             FileService.SaveTemplate(template);
-            var page = ContentTypeBuilder.CreateSimpleContentType("page", "Page", null, false, "Content_", defaultTemplateId: template.Id);
+            var page = ContentTypeBuilder.CreateSimpleContentType("page", "Page", propertyGroupName: "Content_", defaultTemplateId: template.Id);
```

Comment:
I saw that you liked the cleaner version of calling `CreateSimpleContentType()` where you remove redundant params due to being the same as the default values of those parameters. I think this is a good approach. There are several other places **in this file** where we can easily remove the `null` value passed on the method call.

Let me know if you would be up for fixing this small "cosmetic" details 😁

Reply:
Thanks, for sure - have updated.